papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -76,6 +86,16 @@
         return ResponseEntity.ok(sprintMapper.mapToApi(created));
     }
 
+    public ResponseEntity<SprintDto> addSprint(SprintDto sprintDto, Long projectId) {
+        Project project = projectsRestService.getValidProject(projectId);
+
+        if (!project.isAdmin(userContext.getUser())) {
+            throw new ForbiddenAccessException();
+        }
+
+        return addSprint(project, sprintDto);
```

Comment:
Mi się chyba nie podoba rozdzielenie tego tak. Metoda była za długa?

Reply:
W sensie to jest trochę jak na asemblerach, jak skoki były za krótkie i trzeba było robić na kilka razy

Reply:
Mieliśmy circular dependency i testy integracyjne concordiona nie przechodziły, bo beany nie mogły wstać. Rozbiłem powiązania między klasami `...RestReservice`. Ta metoda tutaj jest tylko przeze mnie przeniesiona z `ProjectRestService`, żeby rozwiązać wspomniany problem i jakoś też zebrać w jednym miejscu metody które robią to samo. Zmiana z `modifySprint` to podobna sytuacja

Reply:
No wiem, przy `modifySprint` też mi się nie podoba :stuck_out_tongue:

Reply:
W sensie jak dla mnie oba `modifySprint` powinny być jedną metodą oraz tak samo oba `addSprint`

Reply:
Bo to mi nic nie mówi:
`public ResponseEntity<SprintDto> modifySprint(Long sprintId, SprintDto sprintDto)`
`public ResponseEntity<SprintDto> modifySprint(SprintDto sprintDto, Long projectId, Long sprintId)`

Reply:
Ogarnięto bałagan podobnych metod, robiących to samo z innymi argumentami